### PR TITLE
chore: update amplitude and conditional tracking

### PR DESCRIPTION
**Pull request by alexnm**
A couple of steps taken:
* Amplitude integrated with their official JS package, not through GTM
* GTM + GA tracking removed
* Amp and sentry keys are hardcoded until we merge the other PR with the dynamic env

### CODE REVIEW

**Review by christianalfoni — COMMENTED** [review]
Haha, I know @CompuIves made this function originally 😂 

`isDoNotTrackEnabled`

I am so fascinated how your brain works, why does it not aim for `isTrackEnabled`? 😂 

Anyways, this makes sense to me, though it is not obvious with the code change that it will fix all tracking. Is this used by the tracking effect in Overmind on V1 etc.? 🤔

**Comment by CompuIves** [comment]
It's because it's based on an existing name [image: 😄]

https://en.m.wikipedia.org/wiki/Do_Not_Track

File: packages/common/src/utils/analytics/amplitude.ts [hunk 1]
```diff
@@ -1,129 +1,105 @@
...
-    ) {
-      amplitude.getInstance().setUserId(null);
-      amplitude.getInstance().regenerateDeviceId();
-    }
-  } else {
+  if (!amplitude) {
     debug('[Amplitude] NOT resetting user id because Amplitude is not loaded');
   }
+
+  amplitude.reset();
```
Comment (CompuIves) [inline packages/common/src/utils/analytics/amplitude.ts:57]: Are we sure that we reset the user id fully here? Looking at the previous code (with the explicit null setting), I have a feeling that there's a bigger story here and maybe it requires the clear null setting.

File: packages/common/src/utils/analytics/google.ts [hunk 2]
```diff
@@ -1,13 +0,0 @@
-import { global } from './utils';
-
-export const track = (_, data) => {
-  if (typeof global.ga !== 'undefined') {
-    global.ga('send', data);
```
Comment (CompuIves) [inline packages/common/src/utils/analytics/google.ts:5]: We'll lose Google Analytics from this, but that should be fine, right?

File: packages/app/src/embed/index.html [hunk 3]
```diff
@@ -52,19 +52,6 @@
     rel="icon"
   />
     <title>Embed - CodeSandbox</title>
-    <!-- Google Tag Manager -->
```
Comment (CompuIves) [inline packages/app/src/embed/index.html:55]: Do we need to change anything in `embed/index.js` to make this work properly?

**Review by CompuIves — COMMENTED** [review]
Some small questions!

File: packages/common/src/utils/analytics/google.ts (hunk 2 above)
Comment (alexnm) [reply packages/common/src/utils/analytics/google.ts:5]: Yes, my understanding is that we want to get rid of GA anyway, for privacy concerns

File: packages/common/src/utils/analytics/amplitude.ts (hunk 1 above)
Comment (alexnm) [reply packages/common/src/utils/analytics/amplitude.ts:57]: I was looking at the docs for the browser SDK and the reset does both user reset and device id: https://www.docs.developers.amplitude.com/data/sdks/browser-2/#reset-when-user-logs-out

**Review by CompuIves — APPROVED** [review]

**Pull request merged**